app/src/androidTest/java/org/openobservatory/ooniprobe/client/OONIAPIClientTest.java

```diff
@@ -0,0 +1,67 @@
+package org.openobservatory.ooniprobe.client;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.openobservatory.ooniprobe.AbstractTest;
+import org.openobservatory.ooniprobe.client.callback.GetMeasurementsCallback;
+import org.openobservatory.ooniprobe.model.api.ApiMeasurement;
+
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+public class OONIAPIClientTest extends AbstractTest {
+    private static final String EXISTING_REPORT_ID = "20190113T202156Z_AS327931_CgoC3KbgM6zKajvIIt1AxxybJ1HbjwwWJjsJnlxy9rpcGY54VH";
+    private static final String EXISTING_REPORT_ID_2 = "20190702T000027Z_AS5413_6FT78sjp5qnESDVWlFlm6bfxxwOEqR08ySAwigTF6C8PFCbMsM";
+    private static final String NONEXISTING_REPORT_ID = "EMPTY";
+    private static final String NON_PARSABLE_URL = "https://\t";
+    private static final String JSON_URL = "https://api.ooni.io/api/v1/measurement/temp-id-263478291";
+
+    @Test
+    public void getMeasurementSuccess() {
+        final CountDownLatch signal = new CountDownLatch(1);
+        a.getApiClient().getMeasurement(EXISTING_REPORT_ID, null).enqueue(new GetMeasurementsCallback() {
```

Comment:
I am still struggling to understand where is `a` declared

Reply:
in a file called `AbstractTest` the main class of every test I guess

Reply:
Having a public attribute called `a` seems very bad to me. Usually short names are better for in stack variables than they are for attributes.